### DISCUSSION TITLE
Is it possible to combine multiple listeners into one?

### DISCUSSION THREAD

**Discussion by busslina** [GregoryConrad/rearch-dart#96] [Q&A] [answered]
Something like this:

```dart
_cancelDataModuleListener = cc.listen((use) {
      use(votePersonalCachedCapsule);
      use(voteOverallCachedCapsule);
      forceUpdate();
    });
```

**Comment by busslina** [comment]
I suppose it won't work... but is there any workaround?

**Comment by GregoryConrad** [comment, marked as answer]
What’s the problem there? The listener will get triggered when either of those capsules update.

**Reply by busslina** [reply]
I thought every use call there was blocking, so the second listener couldn't be triggered if the first isn't.


My bad

**Reply by busslina** [reply]
I think it's my poor experience with PR that leads me to this misunderstanding. I suggest you to add an example in the docs with multiple listeners
